Source/Tools/Toolbox/Scene/DebugCameraController.h

```diff
@@ -46,11 +46,14 @@ class URHO3D_TOOLBOX_API DebugCameraController : public LogicComponent
     /// Control camera.
     void Update(float timeStep) override;
 
+    void SetSelection(const ea::vector<WeakPtr<Node>>* selection);
+
 protected:
     /// Current camera speed.
     float speed_ = 2.f;
     /// Current mouse sensitivity.
     float mouseSensitivity_ = 0.1f;
+    const ea::vector<WeakPtr<Node>>* sceneSelection_ = nullptr;
```

Comment:
both `sceneSelection_` and `SetSelection` should have comments as part of our coding style. If you really can't think of a comment or think it is pointless - add empty comment like so:
```cpp
///
const ea::vector<WeakPtr<Node>>* sceneSelection_ = nullptr;
```